### ISSUE TITLE
Migrate to React 15.5.0+

### ISSUE THREAD

**Issue by verekia** [verekia/js-stack-from-scratch#205]
Should be mostly the PropTypes changes:

https://facebook.github.io/react/blog/2017/04/07/react-v15.5.0.html

**Comment by lavebug** [comment]
Will this solve it? https://github.com/reactjs/react-codemod#react-proptypes-to-prop-types

**Comment by lavebug** [comment]
Has any1 done this successfully can you describe the steps, please.

**Comment by pronebird** [comment]
@lavebug it's described at https://facebook.github.io/react/blog/2017/04/07/react-v15.5.0.html

**Comment by lavebug** [comment]
Andrey have you done this migration so there is no warning for all those dependencies in your node_modules? Did you use react-codemod with JSCodeshift? Maybe list steps eli5 don't copy/paste the link from verekia up above.

**Comment by pronebird** [comment]
@lavebug I thought that was what you asked about. 

Shortly, you migrate your code as described in the blog post. For third-party deps you go case-by-case but as far as I know many packages are not yet up to date. I wasn't able to get rid of the warning.
